documentation/specs/src/interoperability/ethereum-bridge/transfers_to_namada.md

```diff
@@ -1,16 +1,18 @@
 # Transferring assets from Ethereum to Namada
 
-In order to facilitate transferring assets from Ethereum to Namada, There 
- will be two internal accounts with associated native validity predicates:
+In order to facilitate transferring assets from Ethereum to Namada, there
+will be two internal accounts with associated native validity predicates:
 
 - `#EthBridge` - Controls the `/eth_msgs/` [storage](ethereum_events_attestation.md#storage)
-- and ledgers of balances
-  for wrapped Ethereum assets (ERC20 tokens) structured in a
-  ["multitoken"](https://github.com/anoma/anoma/issues/1102) hierarchy
-- `#EthBridgeEscrow` which will hold in escrow wrapped Namada tokens which have
-  been sent to Ethereum.
+  and ledger of balances of minted wrapped Ethereum assets (ERC20 tokens) structured in a
+  ["multitoken"](https://github.com/anoma/anoma/issues/1102) hierarchy.
```

Comment:
Ideally the specs shouldn't link to Github issues, which are pretty ephemeral. Do we have multitoken docs somewhere?

Reply:
What about the multitoken section of the specs, actually?